slice/IceGrid/Admin.ice

```diff
@@ -806,99 +783,99 @@ module IceGrid
+        /// @throws ServerNotExistException Thrown when the server doesn't exist.
+        /// @throws NodeUnreachableException Thrown when the node is unreachable.
+        /// @throws DeploymentException Thrown when the server couldn't be deployed on the node.
         FileIterator* openServerLog(string id, string path, int count)
             throws FileNotAvailableException, ServerNotExistException, NodeUnreachableException, DeploymentException;
 
-        /// Open the given server stderr file for reading. The file can be read with the returned file iterator.
-        /// @param id The server id.
+        /// Opens a server stderr file for reading.
+        /// @param id The server ID.
         /// @param count Specifies where to start reading the file. If negative, the file is read from the beginning.
-        /// If 0 or positive, the file is read from the last <code>count</code> lines.
-        /// @return An iterator to read the file. The returned proxy is never null.
-        /// @throws FileNotAvailableException Raised if the file can't be read.
-        /// @throws ServerNotExistException Raised if the server doesn't exist.
-        /// @throws NodeUnreachableException Raised if the node could not be reached.
-        /// @throws DeploymentException Raised if the server couldn't be deployed on the node.
+        /// If 0 or positive, the file is read from the last @p count lines.
+        /// @return An iterator to read the file. This proxy is never null.
+        /// @throws FileNotAvailableException Thrown when the file can't be read.
+        /// @throws ServerNotExistException Thrown when the server doesn't exist.
+        /// @throws NodeUnreachableException Thrown when the node is unreachable.
+        /// @throws DeploymentException Thrown when the server couldn't be deployed on the node.
         FileIterator* openServerStdErr(string id, int count)
             throws FileNotAvailableException, ServerNotExistException, NodeUnreachableException, DeploymentException;
 
-        /// Open the given server stdout file for reading. The file can be read with the returned file iterator.
+        /// Opens a server stdout file for reading.
         /// @param id The server id.
         /// @param count Specifies where to start reading the file. If negative, the file is read from the beginning.
-        /// If 0 or positive, the file is read from the last <code>count</code> lines.
-        /// @return An iterator to read the file. The returned proxy is never null.
-        /// @throws FileNotAvailableException Raised if the file can't be read.
-        /// @throws ServerNotExistException Raised if the server doesn't exist.
-        /// @throws NodeUnreachableException Raised if the node could not be reached.
-        /// @throws DeploymentException Raised if the server couldn't be deployed on the node.
+        /// If 0 or positive, the file is read from the last @p count lines.
```

Comment:
Should we use code blocks for number literals? Like `0`.

Reply:
Replaced by Otherwise.